website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -362,3 +837,209 @@ static SnowflakeIO.CsvMapper<GenericRecord> getCsvMapper() {
            };
 }
 {{< /highlight >}}
+## Cross Language
+### Intro
+Cross-language which is part of [Portability Framework Roadmap](https://beam.apache.org/roadmap/portability/) which aims to provide full interoperability across the Beam ecosystem. From a developer perspective it means the possibility of combining transforms written in different languages(Java/Python/Go). 
+
+Snowflake cross-language implementation is supporting both reading and writing operations for Python programming language. 
+
+Currently, cross-language is supporting only [Apache Flink](https://flink.apache.org/) as a runner in a stable manner but plans are to support all runners. For more information about cross-language please see [multi sdk efforts](https://beam.apache.org/roadmap/connectors-multi-sdk/) and [Beam on top of Flink](https://flink.apache.org/ecosystem/2020/02/22/apache-beam-how-beam-runs-on-top-of-flink.html) articles. 
+
+### Set up 
```

Comment:
This whole part requires rewriting as we don't need any more those steps with docker

Reply:
I'm looking for a volunteer then ;)

Reply:
@purbanow Is there any place with those cross language changed described?

Reply:
Nope

We must remove all those steps and add some new staff.  Maybe just saying that we are supporting cross-language and here is the example how to use it.

Reply:
I will fixed this secton.